README.md

```diff
@@ -1,4 +1,3 @@
-[![Build Status](https://travis-ci.org/MetaModels/attribute_translatedtabletext.svg)](https://travis-ci.org/MetaModels/attribute_translatedtabletext)
 [![Latest Version tagged](http://img.shields.io/github/tag/MetaModels/attribute_translatedtabletext.svg)](https://github.com/MetaModels/attribute_translatedtabletext/tags)
```

Comment:
add
`[![Build Status](https://github.com/MetaModels/attribute_translatedtabletext/actions/workflows/diagnostics.yml/badge.svg)](https://github.com/MetaModels/attribute_translatedtabletext/actions)`

Reply:
see comment

Reply:
Fixed in 02939bc